openwisp_utils/admin_theme/admin.py

```diff
@@ -18,6 +28,74 @@ class OpenwispAdminSite(admin.AdminSite):
     )
     enable_nav_sidebar = False
 
+    def login(self, *args, **kwargs):
+        response = super().login(*args, **kwargs)
+        if (
+            isinstance(response, HttpResponseRedirect)
+            and app_settings.ADMIN_DASHBOARD_VISIBLE
+        ):
+            response = HttpResponseRedirect(reverse('admin:ow_dashboard'))
+        return response
+
+    def dashboard(self, request, extra_context=None):
+        context = {
+            'is_popup': False,
+            'has_permission': True,
+        }
+        schema = copy.deepcopy(DASHBOARD_SCHEMA)
```

Comment:
I'm not sure if the name `DASHBOARD_SCHEMA`, because more than a schema, it is the configuration of the dashboard. Schema could be confused with the concept of the DB schema or JSON schema, what do you think?

Reply:
I agree with you, we can name it `DASHBOARD_CONFIG`.